<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1337</width>
    <height>733</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cockroach Races</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="image1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>110</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/resources/img/second.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>289</x>
      <y>40</y>
      <width>1001</width>
      <height>500</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="start">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>280</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift</family>
      <pointsize>22</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>start</string>
    </property>
   </widget>
   <widget class="QLabel" name="image2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>110</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/resources/img/first.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="image5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>490</y>
      <width>110</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/resources/img/fifth.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="image4">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>110</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/resources/img/fourth.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="image3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>400</y>
      <width>110</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/resources/img/third.gif</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="startGame">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>260</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift</family>
      <pointsize>22</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>start game</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="playerCount">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>80</y>
      <width>51</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>5</number>
    </property>
   </widget>
   <widget class="QLabel" name="l_count">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>110</y>
      <width>551</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift</family>
      <pointsize>28</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Выберите количество игроков</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_player_name">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>100</y>
      <width>831</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift</family>
      <pointsize>28</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="player_name">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>160</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="cockroachName_1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Fast</string>
    </property>
   </widget>
   <widget class="QLabel" name="cockroachName_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Speed</string>
    </property>
   </widget>
   <widget class="QLabel" name="cockroachName_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>380</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Storm</string>
    </property>
   </widget>
   <widget class="QLabel" name="cockroachName_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Lord</string>
    </property>
   </widget>
   <widget class="QLabel" name="cockroachName_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>460</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Leopard</string>
    </property>
   </widget>
   <widget class="QComboBox" name="chooseCockroach">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>220</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="enterAnswer">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>280</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift</family>
      <pointsize>22</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>ok</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="playerSum_1">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>570</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="playerSum_2">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>570</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="playerSum_3">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>570</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="playerSum_4">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>570</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="playerSum_5">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>570</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="playerName_1">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>540</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="playerName_2">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>540</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="playerName_3">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>540</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="playerName_4">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>540</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="playerName_5">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>540</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="placeBetButton">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>610</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift</family>
      <pointsize>22</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>bet</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="betTable">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>291</width>
      <height>498</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Players</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Money</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Bet</string>
     </property>
    </column>
   </widget>
   <widget class="QLCDNumber" name="lcdTimer">
    <property name="geometry">
     <rect>
      <x>1190</x>
      <y>0</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="enterCount">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>280</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift</family>
      <pointsize>22</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>ok</string>
    </property>
   </widget>
   <widget class="QPushButton" name="continueButton">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>610</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift</family>
      <pointsize>22</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>continue</string>
    </property>
   </widget>
   <widget class="QPushButton" name="finishButton">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>610</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift</family>
      <pointsize>22</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>finish</string>
    </property>
   </widget>
   <widget class="QLabel" name="name">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift</family>
      <pointsize>16</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="races">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift</family>
      <pointsize>16</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Races</string>
    </property>
   </widget>
   <widget class="QLabel" name="win">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>20</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift</family>
      <pointsize>16</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Win</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="races_4">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>120</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="races_1">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>230</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="races_2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>330</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="races_3">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>420</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="races_5">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>510</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="win_4">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>120</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="win_1">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>230</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="win_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>330</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="win_3">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>420</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="win_5">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>510</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <zorder>textBrowser</zorder>
   <zorder>image1</zorder>
   <zorder>start</zorder>
   <zorder>image2</zorder>
   <zorder>image5</zorder>
   <zorder>image4</zorder>
   <zorder>image3</zorder>
   <zorder>startGame</zorder>
   <zorder>playerCount</zorder>
   <zorder>l_count</zorder>
   <zorder>l_player_name</zorder>
   <zorder>player_name</zorder>
   <zorder>cockroachName_1</zorder>
   <zorder>cockroachName_2</zorder>
   <zorder>cockroachName_3</zorder>
   <zorder>cockroachName_4</zorder>
   <zorder>cockroachName_5</zorder>
   <zorder>chooseCockroach</zorder>
   <zorder>enterAnswer</zorder>
   <zorder>playerSum_1</zorder>
   <zorder>playerSum_2</zorder>
   <zorder>playerSum_3</zorder>
   <zorder>playerSum_4</zorder>
   <zorder>playerSum_5</zorder>
   <zorder>playerName_1</zorder>
   <zorder>playerName_2</zorder>
   <zorder>playerName_3</zorder>
   <zorder>playerName_4</zorder>
   <zorder>playerName_5</zorder>
   <zorder>placeBetButton</zorder>
   <zorder>betTable</zorder>
   <zorder>lcdTimer</zorder>
   <zorder>enterCount</zorder>
   <zorder>continueButton</zorder>
   <zorder>finishButton</zorder>
   <zorder>name</zorder>
   <zorder>races</zorder>
   <zorder>win</zorder>
   <zorder>races_4</zorder>
   <zorder>races_1</zorder>
   <zorder>races_2</zorder>
   <zorder>races_3</zorder>
   <zorder>races_5</zorder>
   <zorder>win_4</zorder>
   <zorder>win_1</zorder>
   <zorder>win_2</zorder>
   <zorder>win_3</zorder>
   <zorder>win_5</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1337</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCockroach_Races">
    <property name="title">
     <string>Made by Aman</string>
    </property>
   </widget>
   <addaction name="menuCockroach_Races"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
